libraries/standard/json/include/json.h

```diff
@@ -30,7 +30,9 @@ typedef enum
     JSONSuccess,
     JSONIllegalDocument,
     JSONMaxDepthExceeded,
-    JSONNotFound
+    JSONNotFound,
+    JSONNullParameter,
+    JSONBadParameter
```

Comment:
would it make sense to have a `JSON_MAX_ENUM` param?

Reply:
There are no iterators for this enum.